frontend/components/SetupPage/FundYourAgent/components/FinishingSetupModal.tsx

```diff
@@ -0,0 +1,31 @@
+import { LoadingOutlined } from '@ant-design/icons';
+import { Flex, Modal, ModalProps, Spin, Typography } from 'antd';
+
+const { Title, Text } = Typography;
+
+const MODAL_STYLES: ModalProps['styles'] = {
+  content: {
+    width: '452px',
+    minHeight: '264px',
+    padding: '32px',
+    borderRadius: '12px',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+  },
+};
+
+export const FinishingSetupModal = () => (
```

Comment:
can we have a generic modal instead of this one and AgentSetupCompleteModal? the config looks quite the same as in AgentSetupCompleteModal, we can then just simply use one component and pass title, content/children, action?

Reply:
yeah, i was thinking about this, even the agent staking has similar modals.